src/app/api/subscribe/route.ts

```diff
@@ -0,0 +1,49 @@
+import { NextRequest, NextResponse } from 'next/server';
+
+import md5 from 'md5';
+
+export async function POST(request: NextRequest) {
+  const { email } = await request.json();
+
+  if (!email) {
+    return NextResponse.json({ error: 'Email is required' }, { status: 400 });
+  }
```

Comment:
I think we should also check email validation on the server to avoid external api calls in case when someone tries to send data not from our form